### ISSUE TITLE
Nemesis error 5001 during launch

### ISSUE THREAD

**Issue by JiriHodan** [ShikyoKira/Project-New-Reign---Nemesis-Main#567]
Hi,
I followed the install guide on youtube, I have it in skyrim se, data.
I wonder I never get to have to program to function properly.
I would so love to have modern behavior engine, but it just don´t work.
It says: 
Checking engine version
Engine is up to date
Version Code: 9291-994

Initializing behavior generation

Mod Checked 1: nemesis
Mod Checked 2: tkuc
Mod Checked 3: tudm

Mod installed: XPMSE
Mod installed: Nemesis PCEA

PCEA Mod: 0Dser_Animations

ERROR(5001): Invalid format has been detected in "animationsetdatasinglefile.txt". Please contact the template creator or mod author
Project: DefaultFemaleData\DefaultFemale.txt
Header: DragonMountedMRhSolo_ShieldOrTorch.txt

Failed to generate behavior

Thank You very much guys for the help if u can help at all.

Jiri

**Comment by jkalend** [comment]
apparently, it should be fixed by now, but at last for me, the issue is still there even though my first version ever was the newest one (0.84beta). It can be fixed by forcefully replacing the animationsetdatasinglefile.txt with the one that was offered by the author themselves.
[animationsetdatasinglefile.txt](https://github.com/ShikyoKira/Project-New-Reign---Nemesis-Main/files/6950051/animationsetdatasinglefile.txt)

**Comment by JiriHodan** [comment]
Hi, thank You.
There r many subfolders nad animationsdtasinglefiles, but no exact name of
it like: animationsetdatasinglefile  .... where to put it exactly pls.
Thank you very much for the help.
Jiri

ne 8. 8. 2021 v 2:29 odesílatel jkalend ***@***.***> napsal:

**Comment by JiriHodan** [comment]
no, that file is smaller than a basic one, still errors about lines...

út 10. 8. 2021 v 18:35 odesílatel chorche26 hodan ***@***.***>
napsal:

**Comment by JiriHodan** [comment]
I mean this text:


Initializing engine update


ERROR(5020): Invalid format has been detected in
"animationsetdatasinglefile.txt". Please contact the template creator

File: D:\Steam\steamapps\common\Skyrim Special
Edition\Data\meshes\nemesis_animationsetdatasinglefile.txt

Line: 1516


Failed to update engine

st 11. 8. 2021 v 1:01 odesílatel chorche26 hodan ***@***.***>
napsal:

**Comment by jkalend** [comment]
You should be rewriting it in : data/nemesis_engine/temp_behaviors

And updating the engine now should only rewrite the file so you would have to do this again manually

**Comment by jkalend** [comment]
All of this comes from https://github.com/ShikyoKira/Project-New-Reign---Nemesis-Main/issues/155

**Comment by Karerushi** [comment]
Replacing the file does not fix the problem for me.. Any suggestions? I will welcome some helping hands

**Comment by Giant-Dad** [comment]
Hello, i have similar problem, theres one difference tho, i dont have any file in temp_behaviors, only empty folders. When i copy the file from mod creator it still doesn't help and the error still happens. Any tips?
